app/views/games/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Games </h1>
```

Comment:
All files should end with a carriage return for POSIX compliance (one blank line at end of file)

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline